### ISSUE TITLE
Create a long form  guide for Blockgeeks "How to Code a ERC721 Contract"

### ISSUE THREAD

**Issue by bountyone** [BountyOneBounties/Issues#1]
We want you to create the best in-depth guide possible in how to create an ERC721 contract

The guide should match Blockgeeks style and depth as seen here. Blockgeeks.com/guides

Use examples, images and make sure all are sourced and credited. The guide must be original content.

The goal is to have the most comprehensive guide possible for creating an ERC721 token.

**Comment by calchulus** [comment]
Gerald Nash has a great starter that I'd be happy to build upon!
https://medium.com/crypto-currently/the-anatomy-of-erc721-e9db77abfc24

**Comment by thebkr7** [comment]
@calchulus That would be excellent!

**Comment by calchulus** [comment]
Sorry for the lack of update, am still working on this :) Hope to have it by the end of the week at the latest!

**Comment by thebkr7** [comment]
@calchulus We did cancel the bounty due to lack of activity but I will fund another one when you are done to make sure that you get rewarded for your work.
Looking forward to it!

**Comment by thebkr7** [comment]
@calchulus Are you making any progress? If not we'll close this bounty.

**Comment by calchulus** [comment]
Yes! Still working on it! Goal is to be done by mid-late next week?
